schemars_derive/src/lib.rs

```diff
@@ -49,6 +49,14 @@ fn derive_json_schema(
 
     let type_name = &cont.ident;
     let (impl_generics, ty_generics, where_clause) = cont.generics.split_for_impl();
+    let mut where_clause = where_clause.cloned();
+    if let Some(bounds) = cont.serde_attrs.ser_bound() {
```

Comment:
This should probably be factored out somewhere.. possibly replacing `add_trait_bounds`.